dspy/teleprompt/gepa/gepa.py

```diff
@@ -288,9 +288,9 @@ def __init__(
         # Logging
         log_dir: str = None,
         track_stats: bool = False,
-        use_wandb: bool = False,
-        wandb_api_key: str | None = None,
-        wandb_init_kwargs: dict[str, Any] | None = None,
+        use_mlflow: bool = False,
```

Comment:
Perhaps we don't even need the parameters anymore?

The users who configure mlflow globally to use with DSPy will get it automatically?